Update 22.vaccines.md (#862)
greenelab/covid19-review: content/22.vaccines.md

As of December 11, 2020, the United States FDA approved this vaccine under an emergency use authorization [@url:https://www.fda.gov/news-events/press-announcements/fda-takes-key-action-fight-against-covid-19-issuing-emergency-use-authorization-first-covid-19].

### Viral Vector Vaccines
As discussed earlier, delivery and presentation of antigens are key to induce immunity against SARS-CoV2.

> Viral vectors have emerged as a safe and efficient method to deliver foreign substances to induce an efficient immune response [@doi:10.1128/CVI.00298-16].

Reviewer: ```suggestion
Viral vectors have emerged as a safe and efficient method to deliver foreign substances to induce an efficient immune response [@doi:10.1128/CVI.00298-16].
These vaccines use a different virus to deliver genetic material from the target virus of interest.
The vaccine uses the host machinery to construct antigen(s) from the transported genetic material.
```

There are several viral vector vaccines that are available for veterinary use [@doi:10.3390/vaccines8040680].

> While there is currently only one FDA-approved viral vector vaccine for the use in humans against Ebola [doi:10.1016/j.cell.2020.03.011], several Phase I and Phase II clinical trial are ongoing.

Reviewer 2: ```suggestion
While there is currently only one FDA-approved viral vector vaccine for the use in humans against Ebola [doi:10.1016/j.cell.2020.03.011], several Phase I and Phase II clinical trial are ongoing [@doi:10.1128/CVI.00298-16].
```
@nilswellhausen is this where you wanted it?

> Various viral vector platforms including Poxviruses, Adenoviruses, and Vesicular Stomatitis Viruses (VSV) are being developed for treating numerous infectious diseases such as Malaria, Ebola, HIV, and more recently, SARS-CoV2. Here we review the current state of the art for developing viral vector vaccines against SARS-CoV2.

Reviewer 2: ```suggestion
Various viral vector platforms including Poxviruses, Adenoviruses, and Vesicular Stomatitis Viruses (VSV) are being developed for treating numerous infectious diseases such as Malaria, Ebola, HIV, and more recently, SARS-CoV-2.
Here we review the current state of the art for developing viral vector vaccines against SARS-CoV-2.
```

> Various viral vector platforms including Poxviruses, Adenoviruses, and Vesicular Stomatitis Viruses (VSV) are being developed for treating numerous infectious diseases such as Malaria, Ebola, HIV, and more recently, SARS-CoV2. Here we review the current state of the art for developing viral vector vaccines against SARS-CoV2.

Reviewer: ```suggestion
These vaccines are able to induce both an antibody and cellular response, however the response is limited due to the immunogenicity of the viral vector used [@doi:10.1016/j.coi.2011.03.006; @doi:10.1038/mt.2009.130].
Various viral vector platforms including Poxviruses, Adenoviruses, and Vesicular Stomatitis Viruses (VSV) are being developed for treating numerous infectious diseases such as Malaria, Ebola, HIV, and more recently, SARS-CoV2. Here we review the current state of the art for developing viral vector vaccines against SARS-CoV2.
```
